plugins/Monolog/Handler/FailureLogMessageDetector.php

```diff
@@ -32,6 +32,8 @@ public function handle(array $record)
         if ($this->isHandling($record)) {
             $this->hasEncounteredImportantLog = true;
         }
+
+        return false;
```

Comment:
same comment, https://sources.debian.org/src/matomo/5.2.2%2Bdfsg-1/debian/patches/0006-Upgrade-monolog-to-v2.patch/#L73